ui/widgets/resizable.js

```diff
@@ -690,6 +699,50 @@ $.widget( "ui.resizable", $.ui.mouse, {
 		};
 	},
 
+	_calculateAdjustedElementDimensions: function( element ) {
+		var ce = element.get( 0 );
+
+		if ( element.css( "box-sizing" ) !== "content-box" ||
+			( !this._hasScroll( ce ) && !this._hasScroll( ce, "left" ) ) ) {
+				return {
+					height: parseFloat( element.css( "height" ) ),
+					width: parseFloat( element.css( "width" ) )
+				};
+		}
+
+		// Check if CSS inline styles are set and use those (usually from previous resizes)
+		var elWidth = parseFloat( ce.style.width );
+		var elHeight = parseFloat( ce.style.height );
+
+		var paddingBorder = this._getPaddingPlusBorderDimensions( element );
```

Comment:
In this repo, we either use a single `var` (as I described in my comment above) or multiple ones in consecutive lines, not spread within the function. This is because the JS engine will hoist the declaration anyway and we want to make it explicit. You're free to assign the values here, but the declarations should be at the top.

Variables declared without initializers should be the first on the list, preferably all in one line if they fit.